xmppserver/src/main/java/org/jivesoftware/openfire/session/LocalSession.java

```diff
@@ -519,4 +524,35 @@ public String getCipherSuiteName() {
     public final Locale getLanguage() {
         return language;
     }
+
+    /**
+     * Retrieves SoftwareVersion data. This method gives access to temporary Software Version data only. You can
+     * Please see
+     * {@link #setSoftwareVersionData(String, String)}  description for more details.
```

Comment:
You're over-using javadoc annotations here. For simple getters and setters, there is no need to explicitly refer to it with \@link and \@see.